Apps/AdminWebClient/src/Server/Models/UserFeedbackView.cs

```diff
@@ -48,6 +48,11 @@ public UserFeedbackView()
         /// </summary>
         public Guid Id { get; set; }
 
+        /// <summary>
+        /// Gets or sets the related user profile id.
+        /// </summary>
+        public string? UserProfileId { get; set; } = string.Empty;
```

Comment:
should it be nullable?

Reply:
Good question. I made it nullable because the field that it's pulling from in UserFeedback is nullable.

Reply:
That is right. We might need to look into it since I don't think the DB model should have that as a nullable field